API server can't find cert on downgrade to 1.5

**Kubernetes version** (use `kubectl version`): 1.7 alpha client (local mods for e2e tests) downgrading master to 1.5.4 beta


**What happened**:
Upon downgrading the master to 1.5.4, the api server failed to find client CA file.

```console
$ cat kube-apiserver.log 
Flag --address has been deprecated, see --insecure-bind-address instead.
I0307 19:12:19.394891       5 feature_gate.go:189] feature gates: map[ExperimentalCriticalPodAnnotation:true]
F0307 19:12:19.394984       5 universal_validation.go:104] Validate server run options failed: unable to load client CA file: open /etc/srv/kubernetes/ca.crt: no such file or directory
```

```console
$ ls -R /etc/srv/kubernetes/
/etc/srv/kubernetes/:
abac-authz-policy.jsonl  basic_auth.csv  etcd-ca.crt  etcd-peer.crt  etcd-peer.key  known_tokens.csv  kube-controller-manager  kube-scheduler  pki

/etc/srv/kubernetes/kube-controller-manager:
kubeconfig

/etc/srv/kubernetes/kube-scheduler:
kubeconfig

/etc/srv/kubernetes/pki:
apiserver-client.crt  apiserver-client.key  apiserver.crt  apiserver.key  ca-certificates.crt  ca.crt  ca.key  kubelet.crt  kubelet.key  serviceaccount.crt  serviceaccount.key
```

**What you expected to happen**:

I expected the API server to work.


**How to reproduce it** (as minimally and precisely as possible):
Bring up a 1.6 or 1.7 alpha cluster. Checkout a 1.5 release branch. Run `./cluster/gce/upgrade.sh -M v1.5.4-beta.0.33+4f5964699b8e70`. It will hang on the health check.

I'm experimenting with #43070 but I'm having etcd startup issues so i can't yet verify the fix. I'm not sure if it's related to downgrading etcd storage version. This is what I'm seeing:

```console
$ cat /var/log/etcd.log
2017-03-14 10:36:37.182654 I | etcdserver: advertise client URLs = http://127.0.0.1:2379
panic: runtime error: makeslice: len out of range

goroutine 1 [running]:
github.com/coreos/etcd/wal.(*decoder).decode(0xc820199350, 0xc820186178, 0x0, 0x0)
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/wal/decoder.go:55 +0x142
github.com/coreos/etcd/wal.(*WAL).ReadAll(0xc8201476c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/wal/wal.go:238 +0x214
github.com/coreos/etcd/etcdserver.readWAL(0xc820193360, 0x19, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc8201476c0, 0xc820191fb0, 0x5, ...)
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/etcdserver/storage.go:84 +0x227
github.com/coreos/etcd/etcdserver.restartNode(0xc8201380e0, 0x0, 0x7fb35c0ec028, 0xc8200af400, 0x0, 0x0, 0x0, 0x0)
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/etcdserver/raft.go:286 +0xff
github.com/coreos/etcd/etcdserver.NewServer(0xc8201380e0, 0xc820138100, 0x0, 0x0)
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/etcdserver/server.go:288 +0x322d
github.com/coreos/etcd/etcdmain.startEtcd(0xc8200fc6c0, 0x0, 0x0, 0x0)
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/etcdmain/etcd.go:287 +0x1a92
github.com/coreos/etcd/etcdmain.Main()
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/etcdmain/etcd.go:117 +0x1e26
main.main()
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/main.go:29 +0x14

goroutine 12 [chan receive]:
github.com/coreos/etcd/Godeps/_workspace/src/github.com/golang/glog.(*loggingT).flushDaemon(0x11b4240)
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/Godeps/_workspace/src/github.com/golang/glog/glog.go:879 +0x67
created by github.com/coreos/etcd/Godeps/_workspace/src/github.com/golang/glog.init.1
        /home/vagrant/gopath/src/github.com/coreos/etcd/release/etcd/gopath/src/github.com/coreos/etcd/Godeps/_workspace/src/github.com/golang/glog/glog.go:410 +0x297

goroutine 32 [syscall]:
os/signal.loop()
        /usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
        /usr/local/go/src/os/signal/signal_unix.go:28 +0x37
$ docker ps | grep etcd
79ece146e545        gcr.io/google_containers/etcd-empty-dir-cleanup:0.0.1                               "/bin/sh -c 'bash /et"   10 minutes ago      Up 10 minutes                           k8s_etcd-empty-dir-cleanup.3835f5ae_etcd-empty-dir-cleanup-kubernetes-master_kube-system_963aeb20d7afa73be8e792c7678ead97_b535554d
9b3a3c9593cd        gcr.io/google_containers/pause-amd64:3.0                                            "/pause"                 11 minutes ago      Up 10 minutes                           k8s_POD.d8dbe16c_etcd-server-kubernetes-master_kube-system_2f378544cb9c7672404bc9188e01bdce_17b3875b
ea260e588988        gcr.io/google_containers/pause-amd64:3.0                                            "/pause"                 11 minutes ago      Up 10 minutes                           k8s_POD.d8dbe16c_etcd-server-events-kubernetes-master_kube-system_e47ef6c609f377aa9162c615817ac09f_77e0a3e8
07ab7db36f5f        gcr.io/google_containers/pause-amd64:3.0                                            "/pause"                 11 minutes ago      Up 11 minutes                           k8s_POD.d8dbe16c_etcd-empty-dir-cleanup-kubernetes-master_kube-system_963aeb20d7afa73be8e792c7678ead97_e9fecc9c
```

--- a/cluster/gce/configure-vm.sh
+++ b/cluster/gce/configure-vm.sh
@@ -69,7 +69,7 @@ function create-node-pki {
 
   # TODO(mikedanese): remove this when we don't support downgrading to versions
   # < 1.6.
-  ln -s "${CA_CERT_BUNDLE_PATH}" /etc/kubernetes/ca.crt
+  ln -sf "${CA_CERT_BUNDLE_PATH}" /etc/kubernetes/ca.crt
 }
 
 # A hookpoint for setting up local devices

--- a/cluster/gce/gci/configure-helper.sh
+++ b/cluster/gce/gci/configure-helper.sh
@@ -219,7 +219,7 @@ function create-node-pki {
 
   # TODO(mikedanese): remove this when we don't support downgrading to versions
   # < 1.6.
-  ln -s "${CA_CERT_BUNDLE_PATH}" /etc/srv/kubernetes/ca.crt
+  ln -sf "${CA_CERT_BUNDLE_PATH}" /etc/srv/kubernetes/ca.crt
 }
 
 function create-master-pki {
@@ -272,8 +272,8 @@ function create-master-pki {
 
   # TODO(mikedanese): remove this when we don't support downgrading to versions
   # < 1.6.
-  ln -s "${APISERVER_SERVER_CERT_PATH}" /etc/srv/kubernetes/server.key
-  ln -s "${APISERVER_SERVER_CERT_PATH}" /etc/srv/kubernetes/server.cert
+  ln -sf "${APISERVER_SERVER_CERT_PATH}" /etc/srv/kubernetes/server.key
+  ln -sf "${APISERVER_SERVER_CERT_PATH}" /etc/srv/kubernetes/server.cert
 }
 
 # After the first boot and on upgrade, these files exist on the master-pd

